mycore-base/src/main/java/org/mycore/common/content/MCRSourceContent.java

```diff
@@ -117,6 +69,58 @@ public MCRSourceContent(Source source) {
         this.setBaseContent(baseContent);
     }
 
+    private static MCRContent getBaseContent(StreamSource streamSource) {
```

Comment:
Any reason why those getBaseContent() methods are static?

Reply:
probably because they are independent of the state of the object ?

Reply:
True, but it also has no benefit. The methods are never used in any static way.

Reply:
I can accept that, but I highly doubt there is any difference in the real world. The JIT-Compiler is probably optimizing this anyway.
In my opinion such decision should never be based on performance (except there is a real reason for it). I would always choose readability, testability and extensibility over it.
But I guess its ok in this case cause we don't have any of that.

Reply:
These methods should be in StreamSource, JDOMSource...